main.py

```diff
@@ -21,6 +21,8 @@
 INTERVAL = EMA_LONG+1
 NEXT = 1
 
+PRODUCTION = False # added for convenience, all factories eventually created in main and passed to trade_bot
```

Comment:
Can we make an enum to describe the data source? That way we would be able to easily expand the enum if/when we add more data sources.

Reply:
sure, I like this suggestion. Also, I noted early on in one of your initial commits I think you named the raw data files after the currency it represented. I thought we could replicate this and have like BCHUSD_candlesticks.csv & BCHUSD_ticks.csv and on and on